<launch>
<param name="use_sim_time" value="true" />
  <!-- fetch or freight? -->
  <arg name="name" default="fetch" />

  <!-- set topics -->
  <arg name="map_topic" default="map" />
  <arg name="cmd_vel_topic" default="cmd_vel" />
  <arg name="odom_topic" default="odom" />

  <!-- planners, costmaps, etc -->
  <node pkg="move_base" type="move_base" name="move_base" respawn="true" output="screen">
    <rosparam file="$(find fetch_navigation)/config/costmap_common.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find fetch_navigation)/config/$(arg name)/costmap_common.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find fetch_navigation)/config/costmap_common.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find fetch_navigation)/config/$(arg name)/costmap_common.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find fetch_navigation)/config/costmap_local_rwrobot.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find fetch_navigation)/config/$(arg name)/costmap_local.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find fetch_navigation)/config/costmap_global.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find fetch_navigation)/config/$(arg name)/costmap_global.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find fetch_navigation)/config/move_base.yaml" command="load" />
    <rosparam file="$(find fetch_navigation)/config/$(arg name)/move_base.yaml" command="load" />
    <remap from="cmd_vel" to="$(arg cmd_vel_topic)"/>
    <remap from="odom" to="$(arg odom_topic)"/>
    <remap from="map" to="$(arg map_topic)" />
  </node>

  <!-- <node pkg="tf" type="static_transform_publisher" name="base_map" args="0 0 0 0 0 0 /map /odom 100" /> -->

</launch>
